src/elements/name_section.rs

```diff
@@ -97,7 +97,10 @@ impl NameSection {
 					local_names = Some(LocalNameSubsection::deserialize(module, rdr)?);
 				},
 
-				_ => return Err(Error::UnknownNameSubsectionType(subsection_type)),
+				_ => {
+					let mut buf = vec![0; size];
+					rdr.read(&mut buf)?;
+				},
```

Comment:
The unknown subsection is just thrown away, right? Maybe add a comment here explaining that.

Reply:
comment added